pkg/apis/v1beta1/network.go

```diff
@@ -46,6 +48,9 @@ func (n *Network) Validate() []error {
 	if n.Provider != "calico" && n.Provider != "custom" {
 		errors = append(errors, fmt.Errorf("unsupported network provider: %s", n.Provider))
 	}
+	if n.Provider == "custom" && n.DualStack.Enabled && n.Calico.Mode != "bird" {
```

Comment:
```suggestion
	if n.Provider == "calico" && n.DualStack.Enabled && n.Calico.Mode != "bird" {
```
I don't think we want to validate Calico mode when running custom CNI :)

Reply:
That cause to validate calico mode only. Like, if we have calico enabled we want to have dualstack enabled only with "bird" mode